### DISCUSSION TITLE
Suggestion: There should be a button to quit from within the Vorta window

### DISCUSSION THREAD

**Discussion by camdendotlol** [borgbase/vorta#920] [Ideas]
Wasn't sure if this should be a full Issue because of how niche my use-case is, so I'm posting here instead.

I'm on GNOME without extensions, so there's no system tray, meaning the Vorta tray icon is inaccessible. I have Vorta set to run in the background at startup, and I click on the icon in the Applications menu to open the window. When I'm done I click the X, which kicks it back to the background. All that works very well.

But if I want to completely quit Vorta (e.g. after a flatpak update, or if I don't want backups to run for a while), there's no simple way to do that without the tray icon. It's possible to check the "Display background exit dialog" option and it will offer to quit when I click the X, but that's rather inconvenient because I typically use the X to just close the window and leave Vorta running in the background.

I know GNOME's decision to remove the system tray is kind of goofy, but I have a feeling that a simple quit button somewhere in the Vorta window would be a usability improvement for other cases too.

**Comment by m3nu** [comment]
Yes, there is already the "Display background exit dialog" option for that.

There could be another option in that dialog to "Always fully quit when closing the main window".

But I'm not sure we should add so many settings.
